pom.xml

```diff
@@ -25,14 +25,15 @@
 		<tycho.scmUrl>scm:git:https://github.com/jbosstools/jbosstools-integration-tests.git</tycho.scmUrl>
 		<!-- URLs needed to resolve dependencies at build time (see <repositories> below) and at install time (see site/pom.xml#associateSites) -->
 
+		<eclipseName>neon</eclipseName>
 		<!-- predefined RedDeer sites, if you need to change default please redefine reddeer-site property in your test -->
-		<reddeer-master-site>http://download.jboss.org/jbosstools/mars/snapshots/builds/reddeer_master/</reddeer-master-site>
+		<reddeer-master-site>http://download.jboss.org/jbosstools/${eclipseName}/snapshots/builds/reddeer_master/</reddeer-master-site>
```

Comment:
This will need to stay on mars for now. Please change this line before applying to PR.